--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -74,3 +74,11 @@
 if (ENABLE_ENCODER)
   add_subdirectory (enc265)
 endif()
+
+set(libdir ${CMAKE_INSTALL_PREFIX}/lib)
+set(VERSION 1.0.3)
+set(LIBS ${CMAKE_INSTALL_PREFIX}/lib)
+set(includedir ${CMAKE_INSTALL_PREFIX}/include)
+
+configure_file(libde265.pc.in libde265.pc @ONLY)
+install(FILES ${CMAKE_BINARY_DIR}/libde265.pc DESTINATION lib/pkgconfig)
--- a/libde265/de265.h
+++ b/libde265/de265.h
@@ -38,9 +38,9 @@
 
 #if defined(_MSC_VER) && !defined(LIBDE265_STATIC_BUILD)
   #ifdef LIBDE265_EXPORTS
-  #define LIBDE265_API __declspec(dllexport)
+  #define LIBDE265_API
   #else
-  #define LIBDE265_API __declspec(dllimport)
+  #define LIBDE265_API
   #endif
 #elif HAVE_VISIBILITY
   #ifdef LIBDE265_EXPORTS
