--- a/libde265/CMakeLists.txt
+++ b/libde265/CMakeLists.txt
@@ -90,7 +90,7 @@
   add_subdirectory (x86)
 endif()
 
-add_library(${PROJECT_NAME} SHARED ${libde265_sources} ${ENCODER_OBJECTS} ${X86_OBJECTS})
+add_library(${PROJECT_NAME} ${libde265_sources} ${ENCODER_OBJECTS} ${X86_OBJECTS})
 target_link_libraries(${PROJECT_NAME} PRIVATE Threads::Threads)
 
 write_basic_package_version_file(${PROJECT_NAME}ConfigVersion.cmake COMPATIBILITY ExactVersion)
@@ -107,3 +107,4 @@
 
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake DESTINATION
     "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+
--- a/libde265/de265.h
+++ b/libde265/de265.h
@@ -38,9 +38,9 @@
 
 #if defined(_MSC_VER) && !defined(LIBDE265_STATIC_BUILD)
   #ifdef LIBDE265_EXPORTS
-  #define LIBDE265_API __declspec(dllexport)
+  #define LIBDE265_API
   #else
-  #define LIBDE265_API __declspec(dllimport)
+  #define LIBDE265_API
   #endif
 #elif HAVE_VISIBILITY
   #ifdef LIBDE265_EXPORTS
