--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -43,7 +43,11 @@
   if(MSVC)
     set(SUPPORTS_SSE4_1 1)
   else()
-    CHECK_C_COMPILER_FLAG(-msse4.1 SUPPORTS_SSE4_1)
+    set(CMAKE_CXX_FLAGS_TMP ${CMAKE_CXX_FLAGS})
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.1")
+    include (${CMAKE_ROOT}/Modules/CheckCXXSymbolExists.cmake)
+    CHECK_CXX_SYMBOL_EXISTS(__SSE4_1__ "" SUPPORTS_SSE4_1)
+    set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS_TMP})
   endif()
 endif()
 
