diff --git a/libheif/color-conversion/rgb2yuv.cc b/libheif/color-conversion/rgb2yuv.cc
index 6b2df3b..f07aa54 100644
--- a/libheif/color-conversion/rgb2yuv.cc
+++ b/libheif/color-conversion/rgb2yuv.cc
@@ -835,7 +835,7 @@ Op_RGB24_32_to_YCbCr444_GBR::convert_colorspace(const std::shared_ptr<const Heif
     }
   }
 
-  uint8_t* out_cb, * out_cr, * out_y, * out_a;
+  uint8_t* out_cb, * out_cr, * out_y, * out_a = nullptr;
   int out_cb_stride = 0, out_cr_stride = 0, out_y_stride = 0, out_a_stride = 0;
 
   const uint8_t* in_p;
diff --git a/libheif/context.cc b/libheif/context.cc
index 28292a5..ffa2d1d 100644
--- a/libheif/context.cc
+++ b/libheif/context.cc
@@ -100,7 +100,7 @@ static int32_t readvec_signed(const std::vector<uint8_t>& data, int& ptr, int le
   val &= ~high_bit;
 
   if (negative) {
-    return -(high_bit - val);
+    return 0-(high_bit - val);
   }
   else {
     return val;
