--- a/libheif/color-conversion/rgb2yuv.cc
+++ b/libheif/color-conversion/rgb2yuv.cc
@@ -837,7 +837,7 @@
     }
   }
 
-  uint8_t* out_cb, * out_cr, * out_y, * out_a;
+  uint8_t* out_cb, * out_cr, * out_y, * out_a = nullptr;
   int out_cb_stride = 0, out_cr_stride = 0, out_y_stride = 0, out_a_stride = 0;
 
   const uint8_t* in_p;
--- a/libheif/context.cc
+++ b/libheif/context.cc
@@ -100,7 +100,7 @@
   val &= ~high_bit;
 
   if (negative) {
-    return -(high_bit - val);
+    return 0-(high_bit - val);
   }
   else {
     return val;
