--- a/libheif/api/libheif/heif_library.h
+++ b/libheif/api/libheif/heif_library.h
@@ -51,9 +51,9 @@
 
 #if (defined(_WIN32) || defined __CYGWIN__) && !defined(LIBHEIF_STATIC_BUILD)
 #ifdef LIBHEIF_EXPORTS
-#define LIBHEIF_API __declspec(dllexport)
+#define LIBHEIF_API
 #else
-#define LIBHEIF_API __declspec(dllimport)
+#define LIBHEIF_API
 #endif
 #elif defined(HAVE_VISIBILITY) && HAVE_VISIBILITY
 #ifdef LIBHEIF_EXPORTS
