--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,3 +70,13 @@
 add_subdirectory (examples)
 add_subdirectory (libheif)
 add_subdirectory (gdk-pixbuf)
+
+set(libdir ${CMAKE_INSTALL_PREFIX}/lib)
+set(prefix ${CMAKE_INSTALL_PREFIX})
+set(exec_prefix ${CMAKE_INSTALL_PREFIX})
+set(VERSION 1.4.0)
+set(LIBS ${CMAKE_INSTALL_PREFIX}/lib)
+set(includedir ${CMAKE_INSTALL_PREFIX}/include)
+
+configure_file(libheif.pc.in libheif.pc @ONLY)
+install(FILES ${CMAKE_BINARY_DIR}/libheif.pc DESTINATION lib/pkgconfig)
