--- a/libheif/heif_context.cc
+++ b/libheif/heif_context.cc
@@ -97,7 +97,7 @@
   val &= ~high_bit;
 
   if (negative) {
-    return -(high_bit-val);
+    return 0-(high_bit-val);
   }
   else {
     return val;
--- a/libheif/heif_image.cc
+++ b/libheif/heif_image.cc
@@ -637,7 +637,7 @@
     outimg->add_plane(heif_channel_Alpha, m_width, m_height, 8);
   }
 
-  uint8_t *out_cb,*out_cr,*out_y, *out_a;
+  uint8_t *out_cb,*out_cr,*out_y, *out_a = nullptr;
   int out_cb_stride=0, out_cr_stride=0, out_y_stride=0, out_a_stride=0;
 
   const uint8_t *in_p;
